packages/core/src/test/lambda/vue/remoteInvoke/invokeLambda.test.ts

```diff
@@ -190,20 +190,11 @@ describe('RemoteInvokeWebview', () => {
 
             getTestWindow().onDidShowDialog((d) => d.selectItem(fileUri))
 
-            const loggerErrorStub = sinon.stub(getLogger(), 'error')
-
-            try {
-                await assert.rejects(
-                    async () => await remoteInvokeWebview.promptFile(),
-                    new Error('Failed to read selected file')
-                )
-                assert.strictEqual(loggerErrorStub.calledOnce, true)
-                assert.strictEqual(loggerErrorStub.firstCall.args[0], 'readFileSync: Failed to read file at path %s %O')
-                assert.strictEqual(loggerErrorStub.firstCall.args[1], fileUri.fsPath)
-                assert(loggerErrorStub.firstCall.args[2] instanceof Error)
-            } finally {
-                loggerErrorStub.restore()
-            }
+            await assert.rejects(
+                async () => await remoteInvokeWebview.promptFile(),
+                new Error('Failed to read selected file')
+            )
+            assertLogsContain('readFileSync: Failed to read file at path %s %O', true, 'error')
```

Comment:
Test actual state instead of mocks. Saves lots of code, finds actual bugs, and gives meaningful tests instead of tests that break for no reason.